digital_land/cli.py

```diff
@@ -310,7 +316,7 @@ def pipeline_cmd(input_path, output_path, null_path, issue_path, use_patch_callb
         collection,
         Organisation().organisation_uri,
         patch,
-        use_patch_callback,
+        pm,
     )
```

Comment:
No `pm.register(harmoniser)` needed? (Like L255)

Reply:
good spot, this may actually be better in `__init__` of harmoniser, so it always happens. Will change now.